src/confluence/io/__init__.py

```diff
@@ -20,17 +28,28 @@ def guess_format(filename):
     """
     basename, ext = os.path.splitext(filename)
     return {
-        '.xlsx': FileFormat.EXCEL
+        '.xlsx': FileFormat.EXCEL,
+        '.json': FileFormat.JSON,
+        '.csv': FileFormat.CSV,
+        '.txt': FileFormat.TEXT
     }[ext.lower()]
 
 
 def read(filename, **kwds):
+    print('io',kwds)
+    print(kwds['index_col'])
```

Comment:
Replace these print statements with `_logger.debug(...)`. If you want to see it now, someone will probably want to look at that same information later.